comms/storage/transcode/jobs.go

```diff
@@ -23,18 +23,20 @@ import (
 )
 
 type Job struct {
-	ID                string           `json:"id"`
-	Status            string           `json:"status"`
-	CreatedAt         *time.Time       `json:"created_at,omitempty"`
-	StartedAt         *time.Time       `json:"started_at,omitempty"`
-	FinishedAt        *time.Time       `json:"finished_at,omitempty"`
-	TranscodeProgress float64          `json:"transcode_progress,omitempty"`
-	Error             string           `json:"error,omitempty"`
-	Probe             *FFProbeResult   `json:"probe,omitempty"`
-	SourceInfo        *nats.ObjectInfo `json:"source_info,omitempty"`
-	ResultInfo        *nats.ObjectInfo `json:"result_info,omitempty"`
-	TransocdeWorkerID string           `json:"transcode_worker_id,omitempty"`
-	JetstreamSequence int              `json:"jetstream_sequence,omitempty"`
+	ID                string             `json:"id"`
+	Template          string             `json:"template"`
+	Status            string             `json:"status"`
+	CreatedAt         *time.Time         `json:"created_at,omitempty"`
+	StartedAt         *time.Time         `json:"started_at,omitempty"`
+	FinishedAt        *time.Time         `json:"finished_at,omitempty"`
+	TranscodeProgress float64            `json:"transcode_progress,omitempty"`
+	Error             string             `json:"error,omitempty"`
+	Probe             *FFProbeResult     `json:"probe,omitempty"`
+	SourceInfo        *nats.ObjectInfo   `json:"source_info,omitempty"`
+	ResultInfo        *nats.ObjectInfo   `json:"result_info,omitempty"` // deprecated: use Results
```

Comment:
can we just use Results and remove ResultInfo (or maybe that's future cleanup)?

Reply:
Gonna keep for now because there's legacy data in the KV and struct needs this field to be able to unmarshal old data...

can remove soon tho.